database/migrations/2019_02_11_222440_create_arts_table.php

```diff
@@ -16,7 +16,9 @@ public function up()
         Schema::create('arts', function (Blueprint $table) {
             $table->increments('id');
             $table->integer('artist_id')->unsigned();
-            //TODO TM!!!!!!!!!!!!
+            $table->string('title')->nullable();
+            $table->date('publish')->nullable(); //Ferniseringsdato (ikke oprettelsesdato på Wign)
```

Comment:
Hvad med året kunsten er blevet lavet? Mange af billederne har ikke en ferniseringsdato, eller at datoen er mange år efter de er lavet. Taget f.eks. Mette Aaen. Hendes kunster er lavet tidligere, er det så først ved DDU's feninsering af hendes kunster vi sætter "publish" datoen på? Bare en ny kolonne med "year" vil være perfekt

Reply:
Jeg tænker mere på at datoen kan bruges til tjek om må kunsten offentliggøres. Så vi kan modtage kunster som skal holdes hemmeligt indtil bestem dato.
Hvis kunstner, fx. Mette Aaen, har ikke noget ferniseringsdato, kan vi bare sætte datoen til dagsdato eller dato de vælger.

Reply:
Super. Så mangler du blot året for værket. Kunsterne vil gerne have det på deres værknavn

Reply:
Tilføjet